src/coreclr/jit/optcse.cpp

```diff
@@ -485,19 +486,21 @@ unsigned Compiler::optValnumCSE_Index(GenTree* tree, Statement* stmt)
         assert(vnStore->IsVNConstant(vnLibNorm));
         key = vnStore->CoercedConstantValue<size_t>(vnLibNorm);
 
-        // We don't shared small offset constants when we require a reloc
+        // We don't share small offset constants when we require a reloc
+        //
```

Comment:
Maybe explain why not?  Is it a legality thing or a profitability thing?

Reply:
It would be incorrect to CSE,,
You can't eliminate a reloc using a CSE and adding an offset to a different constant.